# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gdmap/07-add-filetypes.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Description: Recognize more file extensions by default
Author: Andres Rodriguez <andreserl@ubuntu.com>
Origin: http://patches.ubuntu.com/g/gdmap/gdmap_0.8.1-1.1ubuntu1.patch

--- gdmap-v1.3.1/src/colors.c.vanilla	2025-06-28 11:21:51.455261938 +0200
+++ gdmap-v1.3.1/src/colors.c	2025-06-28 11:23:41.032979752 +0200
@@ -301,15 +301,14 @@
     Colors = g_list_append(Colors, color_new(0x0f0f, 0xffff, 0xffff, COLOR_MARK2));
     Colors = g_list_append(Colors, color_new(0x2222, 0x4444, 0x0000, COLOR_FOLDER));
 
-    Colors = g_list_append(Colors, color_new_with_exts("#DD0000", "asf avi mpg mpeg wmv"));
-    Colors = g_list_append(Colors, color_new_with_exts("#FF00FF", "exe"));
+    Colors = g_list_append(Colors, color_new_with_exts("#DD0000", "asf avi mpg mpeg wmv mov"));
+    Colors = g_list_append(Colors, color_new_with_exts("#FF00FF", "exe msi deb rpm run"));
     Colors = g_list_append(Colors, color_new_with_exts("#BB00BB", "dll o lo a so"));
-    Colors = g_list_append(Colors, color_new_with_exts("#00AA00", "c cc cpp"));
+    Colors = g_list_append(Colors, color_new_with_exts("#00AA00", "c cc cpp py php php3 php4 php5 js sh bat"));
     Colors = g_list_append(Colors, color_new_with_exts("#00FF00", "h hh"));
-    Colors = g_list_append(Colors, color_new_with_exts("#EEEE00", "eps bmp svg gif png jpg jpeg tif tiff"));
-    Colors = g_list_append(Colors, color_new_with_exts("#00DDDD", "mp3 ogg flac wav"));
-/*   Colors = g_list_append(Colors, color_new_with_exts("#d8def1", "odp ods odt otp ots ott stc stw sxc sxi sxw")); */
-    Colors = g_list_append(Colors, color_new_with_exts("#66aaff", "mdb doc pdf ps ppt txt po xsl xls xml html htm"));
+    Colors = g_list_append(Colors, color_new_with_exts("#EEEE00", "eps bmp svg gif png jpg jpeg tif tiff xpm"));
+    Colors = g_list_append(Colors, color_new_with_exts("#00DDDD", "mp3 ogg flac wav wma m4a"));
+    Colors = g_list_append(Colors, color_new_with_exts("#66aaff", "mdb doc pdf ps ppt txt po xsl xls xml html htm odp ods otp ots ott stc stw sxc sxi sxw hlp tex"));
     Colors = g_list_append(Colors, color_new_with_exts("#0000AA", "zip gz tgz rar bz2 7z"));
 
 }
